recipes/diligent-core/all/conanfile.py

```diff
@@ -83,14 +87,14 @@ def config_options(self):
             del self.options.fPIC
 
     def _patch_sources(self):
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            tools.patch(**patch)
+        patches.apply_conandata_patches(self)
 
     def build_requirements(self):
-        self.build_requires("cmake/3.22.0")
+        self.tool_requires("cmake/3.24.0")
 
     def requirements(self):
         self.requires("opengl/system")
+        self.requires("wayland/1.21.0", override=True)
```

Comment:
```suggestion
        self.requires("wayland/1.21.0")
```
Not allowed. If have any conflict, it should be solved by aligning dependencies version